app/controllers/application_controller.rb

```diff
@@ -31,6 +31,13 @@ def authorize_user!
     return false
   end
 
+  def authorize_instructor!
+    return true if is_an_instructor? || has_api_token?
```

Comment:
students query their info with an access_token as well. see https://github.com/wdidc/status/blob/7a8063f095b7bdebf654e7ad730278334215c0c8/js/app.js#L73 does this affect that?